lms/djangoapps/teams/views.py

```diff
@@ -330,7 +330,13 @@ def get(self, request):
             result_filter.update({'topic_id': request.QUERY_PARAMS['topic_id']})
 
         if text_search and CourseTeamIndexer.search_is_enabled():
-            search_engine = CourseTeamIndexer.engine()
+            try:
+                search_engine = CourseTeamIndexer.engine()
+            except ElasticSearchConnectionError:
+                return Response(
```

Comment:
I'm not sure what the current plan is wrt the bug on stage, but this seems to substitute a 500 for a 400, and I'd rather the user see something useful. As it stands this would cause the search field to be mysteriously unresponsive, though at least it won't take down other teams endpoints.